common/src/main/java/tachyon/worker/WorkerClient.java

```diff
@@ -114,16 +114,18 @@ public synchronized void accessBlock(long blockId) throws IOException {
   }
 
   /**
-   * Notifies the worker that the checkpoint file of the file has been added.
+   * Notifies the worker that a file has been persisted.
    *
-   * @param fileId The id of the checkpointed file
+   * @param fileId the file id
+   * @param nonce nonce a nonce used for temporary file creation
+   * @param path the UFS path of the file
```

Comment:
NITs on documentation:
- "Notifies the worker that a file has been persisted." in the temp path, correct?
- param path is the real ufs path (after file complete) rather than the temp file path, correct?